src_assets/common/assets/web/config.html

```diff
@@ -1140,6 +1163,7 @@ <h2 class="accordion-header">
     "encoder": "",
     "fps": "[10,30,60,90,120]",
     "gamepad": "auto",
+    "ds4_back_as_touchpad": "enabled",
```

Comment:
Alphabetical please.

Reply:
Thanks, and I now see others that are out of order. I'll correct them in a separate PR.